go-controller/pkg/crd/dnsobject/v1/doc.go

```diff
@@ -0,0 +1,4 @@
+// Package v1 contains API Schema definitions for the network v1 API group
```

Comment:
nit: convention is to put api types in `/pkg/api`

Reply:
all the other crds are structured in the same way as this one. If we want to change the crd structure to include a `/pkg/api` can I make a separate git issue for after this merges?

Reply:
aha, missed that. Probably not worth changing now.